docs/en/observability/slo-troubleshoot.asciidoc

```diff
@@ -0,0 +1,404 @@
+
+The rollup documents are stored in an index named `.slo-observability.sli-v3` (index split per month through an ingest pipeline) while summary documents are stored in `.slo-observability.summary-v3`.
+
+Each time an SLO is updated, a new transform is created using the latest definition. The transform ID is generated by combining the SLO id and the SLO revision, following the format: `slo-{slo.id}-{slo.revision}`.
+
+Ensuring that transforms are functioning correctly and that the cluster is healthy is crucial for maintaining accurate and reliable SLOs.
+
+(TBD: explain also the main pipelines associated with SLOs and their objectives?)
+(TBD: anything to add about index templates or other indices being used?)
+
+[discrete]
+[[slo-common-problems]]
+== Common problems
+
+One of the common issues with SLOs arises when there are underlying problems in the cluster, such as unavailable shards or failed transforms. Since SLOs rely on transforms to aggregate and process data, any failure or misconfiguration in these components can lead to inaccurate or incomplete SLO calculations. Additionally, unavailable shards can affect the data retrieval process, further complicating the reliability of SLO metrics.
+
+[discrete]
+[[slo-no-transform-ingest-node]]
+=== No transform or ingest nodes
+
+Since SLOs depend on both {ref}/ingest.html[ingest pipelines] and {ref}/transforms.html[transforms] to process the data, it's essential to ensure that the cluster has nodes with the appropriate {ref}/modules-node.html#node-roles[roles].
+
+Ensure the cluster includes one or more nodes with both `ingest` and `transform` roles (they can co-exist or be distributed across separate nodes), to support the data processing and transformations required for SLOs to function properly.
+
+[discrete]
+[[slo-transform-unhealthy]]
+=== Unhealthy transforms
+
+When working with SLOs, ensuring that the associated transforms function correctly is crucial. Transforms are responsible for generating the data needed for SLOs, and typically, two transforms are created for each SLO. If you notice that your SLOs are not displaying the expected data, it's time to check the health of these associated transforms.
+
+{kib} shows the following message when any of the associated transforms is in an unexpected state:
+
+* `"The following transform is an unhealthy state"`, followed by a list of transforms, as shown in the picture:
+(TBD: add screenshot of how the unhealthy transform report looks like, the warning introduced in 8.15)
+
+In the case of reported **unhealthy transforms**, refer to the {ref}/transform-troubleshooting.html[troubleshooting transforms] documentation for detailed guidance on diagnosing and resolving transform-related issues.
+
+Additionally, it is recommended to perform the following **transform checks**:
+
+* Ensure the needed transforms for the SLOs haven't been deleted or stopped.
```

Comment:
Good point, the UI is not obvious as it will only says "unhealthy transform" even if it doesn't exist.
That's why I have added a generic `check that the transform hasn't been deleted or stopped`. We could change it to something like `check that the transform exists and it's not stopped`.

But anyway it wouldn't change much in my opinion. It's true that it's a bit vague, but the entire doc is a bit vague because we don't have exact issues with reasons to talk about.